beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -26,6 +26,60 @@ use super::{
     Worker,
 };
 
+/// An attestation that has been validated by the `BeaconChain`.
+///
+/// Since this struct implements `beacon_chain::VerifiedAttestation`, it would be a logic error to
+/// construct this from components which have not passed `BeaconChain` validation.
+struct VerifiedUnaggregate<T: BeaconChainTypes> {
+    attestation: Attestation<T::EthSpec>,
```

Comment:
```suggestion
    attestation: Box<Attestation<T::EthSpec>>,
```

It seems that this struct exists so that we can retain ownership over the inner message, ultimately so we have the option of passing it to `handle_attestation_verification_failure`. Seeing as we box in that case, we may as well avoid unboxing (from the `GossipAttestationPackage`) and reboxing.

I implemented this here: https://github.com/michaelsproul/lighthouse/commit/fe3921d50d7f7367652e9419ce7842deb1a05f84

Might help cut down on memory fragmentation, but probably only marginally